app/src/main/java/com/example/plantsapp/presentation/ui/authentication/AuthFragment.kt

```diff
@@ -34,7 +35,7 @@ class AuthFragment : Fragment(R.layout.fragment_auth) {
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
         super.onViewCreated(view, savedInstanceState)
 
-        activity?.setTitle(R.string.title_auth_screen)
+        (activity as AppCompatActivity).supportActionBar?.hide()
```

Comment:
FYI: Optionally it could be moved to activity to store this logic somewhere in one place, but anyway it is not a problem and not needed to be done right now

Reply:
Or you can move It to hide/show this in current fragment (like on destroy view)
Yes, I think you should do it